css/v202209/symbiota/base.css

```diff
@@ -509,6 +509,10 @@ button.icon-button {
   border-color: buttonborder;
 }
 
+button.icon-button > svg {
```

Comment:
Ask Ed where this new styling should go, because I think after this PR: https://github.com/BioKIC/Symbiota/pull/1514, we won't want to add to this file?